### PR TITLE
#5441: adds translations for Sanskrit, Kannada & Telugu

### PR DESCRIPTION
**Pull request by prnam**
This is language support that intends to close the issue #5441, #5443 & #5445  that was raised. I've don't the translations that is accurate to the best of my knowledge. 

If there is any disagreement by folks on it, please feel free to review this PR or raise new change request (if already approved) as you notice so that we can improve it as we move ahead.

### CODE REVIEW

**Comment by prnam** [comment]
@squidfunk - as discussed earlier in thread, I have added 3 of many Indian languages. I hope at some point rest will be covered by folks who have expertise in writing and framing the sentence. As of now I don't have them and in case I do in future and can contribute, will sure do.

File: material/partials/languages/kn.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    "header": "ಮೇಲ್ಟಿಪ್ಪಣಿ",
+    "meta.comments": "ಪ್ರತಿಕ್ರಿಯೆಗಳು",
+    "meta.source": "ಮೂಲ",
+    "nav": "ಸಂಚರಣೆ",
+    "readtime.one": "ಓದಲು ೧ ನಿಮಿಷ ತೆಗೆದುಕೊಳ್ಳುತ್ತದೆ",
+    "readtime.other": "ಓದಲು # ನಿಮಿಷಗಳನ್ನು ತೆಗೆದುಕೊಳ್ಳುತ್ತದೆ",
+    "rss.created": "ಆರ್ಎಸ್ಎಸ್ ಸೇವೆ",
+    "rss.updated": "ಆರ್ಎಸ್ಎಸ್ ಸೇವೆಯಿಂದ ಇತ್ತೀಚಿನ ನವೀಕರಣ",
+    "search": "ಹುಡುಕಿ",
+    "search.config.lang": "kn",
```
Comment (squidfunk) [inline material/partials/languages/kn.html:35]: Have you actually tested whether Material for MkDocs works with those values? [`lunr-languages`](https://github.com/MihaiValentin/lunr-languages) has _very_ limited support for Indian languages. My guess is that there will be errors in the console, no search, and other effects.

**Review by squidfunk — CHANGES_REQUESTED** [review]
Many thanks! Some minor changes are necessary. Also please make sure to read the [translation section in our contributing guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md#missing-translations).

File: material/partials/languages/kn.html (hunk 1 above)
Comment (prnam) [reply material/partials/languages/kn.html:35]: Please accept my apologies. I misunderstood that it was just as simple as adding the jinja file, build the project and check if the languages is reflected by changing theme language at `mkdocs.yml`.

Anyway, let me look at this and try to make necessary changes.

**Comment by prnam** [comment]
> Many thanks! Some minor changes are necessary. Also please make sure to read the [translation section in our contributing guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md#missing-translations).

Did not understand what are you hinting at @squidfunk. What did I miss 🤔?

File: material/partials/languages/kn.html (hunk 1 above)
Comment (prnam) [reply material/partials/languages/kn.html:35]: In my opinion there must be short info of `lunr-languages` in contributions guide's missing translation section.

Comment (prnam) [reply material/partials/languages/kn.html:35]: Raised PR to that would add the language support on [lunr-languages](https://github.com/MihaiValentin/lunr-languages/pull/100).

**Comment by squidfunk** [comment]
Thanks for your work again! I've made the necessary adjustments in #5458 (retaining your original commits of course). I'll try to set aside some time to merge #5320, which will include a detailed guide how to provide new translations.

**Pull request closed without merging**